### PR TITLE
Add new options to IncludesPgce

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

Moar qualifications

### Changes proposed in this pull request

IncludesPgce covers all qualifications option

### CODE REVIEW

File: src/domain/Models/Enums/IncludesPgce.cs [hunk 1]
```diff
@@ -2,8 +2,12 @@ namespace GovUk.Education.SearchAndCompare.Domain.Models.Enums
 {
     public enum IncludesPgce
```
Comment (timabell) [inline src/domain/Models/Enums/IncludesPgce.cs]: We should refactor this to update the names in the news context. Tech debt card...

Comment (timabell) [reply src/domain/Models/Enums/IncludesPgce.cs]: https://trello.com/c/uHJsZLD2/421-rename-includespgce-enum

File: src/domain/Models/Enums/IncludesPgce.cs [hunk 2]
```diff
@@ -2,8 +2,12 @@ namespace GovUk.Education.SearchAndCompare.Domain.Models.Enums
...
     {
-        No = 0,
-        Yes = 1,
-        Optional = 2
+        No = 0, // QtsOnly
+        Yes = 1, // QtsWithPgce
+        QtsWithOpionalPgce = 2,
```
Comment (tvararu) [inline src/domain/Models/Enums/IncludesPgce.cs]: Opional

Comment (timabell) [reply src/domain/Models/Enums/IncludesPgce.cs]: Add that to the trello card above...

**Comment by timabell** [comment]
[PGDE and QTLS courses](https://trello.com/c/MYv50sko/413-pgde-and-qtls-courses)

**Pull request merged**